src/main/java/io/gravitee/repository/jdbc/management/JdbcTicketRepository.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.repository.jdbc.management;
+
+import io.gravitee.common.data.domain.Page;
+import io.gravitee.repository.exceptions.TechnicalException;
+import io.gravitee.repository.jdbc.orm.JdbcObjectMapper;
+import io.gravitee.repository.management.api.TicketRepository;
+import io.gravitee.repository.management.api.search.Order;
+import io.gravitee.repository.management.api.search.Pageable;
+import io.gravitee.repository.management.api.search.Sortable;
+import io.gravitee.repository.management.api.search.TicketCriteria;
+import io.gravitee.repository.management.model.Ticket;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Repository;
+
+import java.sql.PreparedStatement;
+import java.sql.Types;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+import static io.gravitee.repository.jdbc.common.AbstractJdbcRepositoryConfiguration.escapeReservedWord;
+
+/**
+ */
+@Repository
+public class JdbcTicketRepository extends JdbcAbstractCrudRepository<Ticket, String> implements TicketRepository {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(JdbcTicketRepository.class);
+
+    private static final String SELECT_ESCAPED_TICKET_TABLE_NAME = "select * from " + escapeReservedWord("tickets");
```

Comment:
Is it necessary to escape the word "tickets" ?

Reply:
resolved